src/screens/Login/GenerateMnemonic/GenerateMnemonicContainer.js

```diff
@@ -4,29 +4,34 @@
  */
 
 import React, { PureComponent } from 'react'
-import { generateMnemonic } from '@chronobank/bitcoin/utils'
+import { getPrivateKeyByMnemonic, generateMnemonic } from '@chronobank/ethereum/utils'
 import PropTypes from 'prop-types'
 import GenerateMnemonic from './GenerateMnemonic'
 
 class GenerateMnemonicContainer extends PureComponent {
 
   state = {
     mnemonic: '',
+    privateKey: '',
+    address: '',
   }
 
   componentDidMount () {
+    const {mnemonic} = this.state
     generateMnemonic()
       .then((resolve) => {
-        this.setState({ mnemonic: resolve })
+        this.setState({ mnemonic: resolve }, () => {
```

Comment:
We should not generate address here.
Address will be generated in scope of 'login' thunk.